build_connectors.sh

```diff
@@ -34,7 +34,7 @@ case "$(uname -m)" in
 esac
 
 # Check if the musl linker is installed
-"$muslLinker" --version > /dev/null 2>&1 || { echo >&2 "ERROR - requires '$muslLinker' linker for compile"; exit 1; }
+# "$muslLinker" --version > /dev/null 2>&1 || { echo >&2 "ERROR - requires '$muslLinker' linker for compile"; exit 1; }
```

Comment:
Hmm, so you don't need the muslLinker installed to build the docker images?

Reply:
you do, but what it's called exactly is platform dependent